AsyncDisplayKit/ASImageNode+AnimatedImage.mm

```diff
@@ -85,6 +85,23 @@ - (void)coverImageCompleted:(UIImage *)coverImage
   }
 }
 
+- (NSString *)animatedImageRunLoopMode
+{
+  ASDN::MutexLocker l(_displayLinkLock);
+  return _animatedImageRunLoopMode;
+}
+
+- (void)setAnimatedImageRunLoopMode:(NSString *)runLoopMode
+{
+  ASDN::MutexLocker l(_displayLinkLock);
+
+  if (_displayLink != nil) {
+    [_displayLink removeFromRunLoop:[NSRunLoop mainRunLoop] forMode:_animatedImageRunLoopMode];
+    [_displayLink addToRunLoop:[NSRunLoop mainRunLoop] forMode:runLoopMode];
```

Comment:
Looks good to me! I don't have a problem changing the default runloop mode, the call should be pretty cheap (though on file bounds it might be a bit more expensive as it has to open and memory map the next file. Might be worth testing on a lower end device though?